src/services/gpt.js

```diff
@@ -99,7 +99,7 @@ export function dfpSettings(handleSlotRenderEnded) {
 export function determineSlotName(dfpCode, slotName) {
   const slotOverride = expandQueryString('adslot');
   if (slotOverride && (slotOverride !== '' || slotOverride !== null)) {
-    return `${dfpCode}/${slotOverride}`;
+    return `/${dfpCode}/${slotOverride}`;
```

Comment:
Was this broken before or was GoogleTag fixing this for us?

Reply:
I think it actually doesn't matter if you have the slash or not. But if you use the full slot name as the adUnit then it matters. So since it looks the same both ways, I decided to be safe and add it.